set.mm

```diff
@@ -8056,6 +8067,31 @@ an illustration of the conservativity of definitions (definitions do not
     ( wo wa andir andi orbi12i bitri ) ABECDEZFAKFZBKFZEACFADFEZBCFBDFEZEABKGLN
     MOACDHBCDHIJ $.
 
+  $( Obsolete proof of ~ annotanannot as of 1-Apr-2022.  (Contributed by AV,
+     8-Mar-2022.)  (Proof modification is discouraged.)
+     (New usage is discouraged.) $)
+  annotanannotOLD $p |- ( ( ph /\ -. ( ph /\ ps ) ) <-> ( ph /\ -. ps ) ) $=
+    ( wa wn wo ianor anbi2i andi pm3.24 pm2.21i id jaoi olc impbii 3bitri ) AAB
+    CDZCAADZBDZEZCAQCZARCZEZUAPSAABFGAQRHUBUATUAUATUAAIJUAKLUATMNO $.
+
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Miscellaneous theorems of propositional calculus
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+  This subsection gathers miscellaneous theorems of propositional calculus,
+  including proofs by cases and other theorems with negated wffs.
+
+$)
+
+$( Theorems relating XOR to conjunction or dsjunction. $)
+
+  $( Theorem *5.16 of [WhiteheadRussell] p. 124.  (Contributed by NM,
+     3-Jan-2005.)  (Proof shortened by Wolf Lammen, 17-Oct-2013.) $)
+  pm5.16 $p |- -. ( ( ph <-> ps ) /\ ( ph <-> -. ps ) ) $=
+    ( wb wn wi wa pm5.18 biimpi imnan mpbi ) ABCZABDCZDZEKLFDKMABGHKLIJ $.
```

Comment:
this does not use df-or

Reply:
This is a compromise.  The XOR related theorems were all in one place, and I want to keep it like that.  The other option is to put this theorem to the NAND based ones, but that is not my preference, others might think different.

Reply:
This should not block the merge, so you can merge now.